src/multidim/setdiff.jl

```diff
@@ -105,3 +105,25 @@ end
 #
 #     return results_list
 # end
+
+import IntervalArithmetic.hull
+doc"""
+    hull(a, b)
+
+Returns the `box hull` of the IntervalBoxes `a` and `b`, considered as
+(extended) $N$ dimensional sets of real numbers, i.e. the smallest region that contains
+all of `a` and `b`.
+"""
+function hull(A::IntervalBox{N,T}, B::IntervalBox{N,T}) where {N,T}
+    result_list = Vector{Interval{T}}()
+
+    global_lo = convert(T,0)
+    global_hi = convert(T,0)
+
+    for dimension in 1:N
```

Comment:
In this loop, you are calculating the `hull` of each pair of intervals, so it can be just written as `hull(A[i], B[i])`. I would use `i` or `n` instead of `dimension` for the loop variable just for readability.